tools/walletextension/frontend/src/components/head-seo.tsx

```diff
@@ -25,9 +25,9 @@ const HeadSeo = ({
         signature="_vd3udx2g2hfn9zclob5cat43b94q7fyk"
       ></meta>
       {/* SECURITY: to prevent the page from being loaded in an iFrame */}
-      <meta http-equiv="X-Frame-Options" content="deny"></meta>
+      <meta httpEquiv="X-Frame-Options" content="deny"></meta>
```

Comment:
<img width="415" alt="image" src="https://github.com/ten-protocol/go-ten/assets/4050716/8d7c2e6e-2190-404a-8539-3251e09f9774">

I stil get those errors in console.